client.go

```diff
@@ -92,7 +92,9 @@ func Connect(conf ConnConf) (*Conn, error) {
 
 	err := c.wsConnect()
 	if err != nil {
-		return nil, c.error("Unable to connect to Exasol: %s", err)
+		err = fmt.Errorf("unable to connect to Exasol: %w", err)
+		c.logUnsupressedError(err)
+		return nil, err
```

Comment:
Hi Andreas - Thanks for the PR.  Is it necessary to break up this line? Could it just be `return nil, c.error("Unable to connect to Exasol: %w", err)` ?

Reply:
It is not necessary. That being said - it feels very unidiomatic to me to call a function without `f` suffix in Go, which does formatting ;)